javascript/tool/js/rich-map.js

```diff
@@ -9,187 +9,197 @@ var map = new mapboxgl.Map({
+      "id": "neighborhood",
+      "type": "line",
+      "source": "neighborhood",
+      layout: {
+        visibility: 'none'
+      },
+      paint: {
+        "line-color": "#0D5C7D",
+        "line-width": 1
+      }
+    });
 
-  map.addSource("zillow", {
-    "type": "geojson",
-    "data": "data/zillow.geojson"
-  });
+    map.addSource("ward", {
+      "type": "geojson",
+      "data": "data/ward.geojson"
+    });
 
-  map.addLayer({
-		"id": "zillow",
-		"type": "line",
-		"source": "zillow",
-		layout: {
-		  visibility: 'none'
-		},
-		paint: {
-		  "line-color": "#57CABD",
-		  "line-width": 1
-		}
-  });
-  //zillow color 57CABD
+    map.addLayer({
+      "id": "ward",
+      "type": "line",
+      "source": "ward",
+      layout: {
+        visibility: 'none'
+      },
+      paint: {
+        "line-color": "#002D61",
+        "line-width": 1
+      }
+    });
 
-  map.addSource("project", {
-    "type": "geojson",
-    "data": "data/project.geojson"
-  });
+    map.addSource("zillow", {
+      "type": "geojson",
+      "data": "data/zillow.geojson"
+    });
 
-  map.addLayer({
-    'id': 'project',
-    'type': 'circle',
-    'source': 'project',
-    'paint': {
-      // make circles larger as the user zooms from z12 to z22
-      'circle-radius': {
-        'base': 1.75,
-        'stops': [[12, 3], [22, 180]]
+    map.addLayer({
+      "id": "zillow",
+      "type": "line",
+      "source": "zillow",
+      layout: {
+        visibility: 'none'
       },
-      // color circles by ethnicity, using data-driven styles
-      'circle-color': {
-        property: 'Category_Code',
-        type: 'categorical',
-        stops: [
-					['1 - At-Risk or Flagged for Follow-up', '#f03b20'],
-					['2 - Expiring Subsidy', '#fecc5c'],
-					['3 - Recent Failing REAC Score', '#fd8d3c'],
-					['4 - More Info Needed', '#A9A9A9'],
-					['5 - Other Subsidized Property', '#A9A9A9'],
-					['6 - Lost Rental', '#bd0026']
-        ]
+      paint: {
+        "line-color": "#57CABD",
+        "line-width": 1
       }
-    }
-  });
+    });
+    //zillow color 57CABD
+    
+    console.log('app in rich-map.js', app);
+    console.log('app.dataCollection.project.geoJSON()', app.dataCollection.project.geoJSON());
+    
+    map.addSource("project", {
+      "type": "geojson",
+      "data": app.dataCollection.project.geoJSON('proj_lon', 'proj_lat')
+    });
+    
+    console.log(map.getSource('project'));
+
+    map.addLayer({
+      'id': 'project',
+      'type': 'circle',
+      'source': 'project',
+      'paint': {
+        // make circles larger as the user zooms from z12 to z22
+        'circle-radius': {
+          'base': 1.75,
+          'stops': [[12, 3], [22, 180]]
+        },
+        // color circles by ethnicity, using data-driven styles
+        'circle-color': {
+          property: 'category_code',
+          type: 'categorical',
+          stops: [
+            ['1 - At-Risk or Flagged for Follow-up', '#f03b20'],
```

Comment:
After Monday's demo when we add other point-based sources, we'll want to find a way to transition this to be a generic categorical color map that can accomodate any other point data source, or to store these type of settings on a per-data set basis so that things get consistently represented across multiple pages of the app. Just flagging for later discussion.